shell/src/main/java/alluxio/shell/command/TestCommand.java

```diff
@@ -0,0 +1,192 @@
+    return "test";
+  }
+
+  @Override
+  public int getNumOfArgs() {
+    return 1;
+  }
+
+  @Override
+  protected Options getOptions() {
+    return new Options()
+        .addOption(DIR_OPTION)
+        .addOption(FILE_OPTION)
+        .addOption(PATH_EXIST_OPTION)
+        .addOption(DIR_NOT_EMPTY_OPTION)
+        .addOption(FILE_ZERO_LENGTH_OPTION);
+  }
+
+  private void printResult(boolean testResult) {
+    if (testResult) {
+      System.out.println(0);
+    } else {
+      System.out.println(1);
+    }
+  }
+
+  /**
+   * Tests whether the path is a directory.
+   *
+   * @param status the {@link URIStatus} status as the input of the command
+   * @return true if the path is a directory or false if it is not a directory
+   */
+  private boolean isDir(URIStatus status) {
+    return status.isFolder();
+  }
+
+  /**
+   * Tests whether the path is a file.
+   *
+   * @param status the {@link URIStatus} status as the input of the command
+   * @return true if the path is a file or false if it is not a file
+   */
+  private boolean isFile(URIStatus status) {
+    return !status.isFolder();
+  }
+
+  /**
+   * Tests whether the directory is not empty.
+   *
+   * @param status the {@link URIStatus} status as the input of the command
+   * @return true if the directory is not empty or false if it is empty
+   */
+  private boolean isNonEmptyDir(URIStatus status) {
+    return status.isFolder() && status.getLength() > 0;
+  }
+
+  /**
+   * Tests whether the file is zero length.
+   *
+   * @param status the {@link URIStatus} status as the input of the command
+   * @return true if the file is zero length or false if it is not zero length
+   */
+  private boolean isZeroLengthFile(URIStatus status) {
+    return !status.isFolder() && status.getLength() == 0;
+  }
+
+  @Override
+  public void run(CommandLine cl) throws AlluxioException, IOException {
+    if (cl.getOptions().length > 1) {
+      System.out.println("test takes only 1 option. Usage: test [-d|-f|-e|-s|-z] <path>");
+      return;
+    }
+    String[] args = cl.getArgs();
+    AlluxioURI path = new AlluxioURI(args[0]);
+    try {
+      URIStatus status = mFileSystem.getStatus(path);
+      boolean testResult = false;
+      if (cl.hasOption("d")) {
+        if (isDir(status)) {
+          testResult = true;
+        }
+      } else if (cl.hasOption("f")) {
+        if (isFile(status)) {
+          testResult = true;
+        }
+      } else if (cl.hasOption("e")) {
+        testResult = true;
+      } else if (cl.hasOption("s")) {
+        if (isNonEmptyDir(status)) {
+          testResult = true;
+        }
+      } else if (cl.hasOption("z")) {
+        if (isZeroLengthFile(status)) {
+          testResult = true;
+        }
+      } else {
+        System.out.println("test takes 1 option, not 0. Usage: test [-d|-f|-e|-s|-z] <path>");
+        return;
+      }
+      printResult(testResult);
+    } catch (AlluxioException | IOException e) {
+      System.out.println(1);
```

Comment:
Sorry for the confusion. When we say return 0 / 1 in the JIRA, we actually mean set the system return value to be 0 / 1. In terms of Java code, you simply call `return 0` or `return 1`, instead of printing them